.docker/qgis.dockerfile

```diff
@@ -1,7 +1,7 @@
 
 # see https://docs.docker.com/docker-cloud/builds/advanced/
 # using ARG in FROM requires min v17.05.0-ce
-ARG DOCKER_TAG=latest
+ARG DOCKER_TAG=release-3_10
```

Comment:
why?

Reply:
Just because this is in the release-3_10 branch so it's odd that it should ever be "latest" ? Can omit this commit if you prefer.